woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php

```diff
@@ -584,6 +584,35 @@ public function update_tokens( $user_id, $tokens, $environment_id = null ) {
 	}
 
 
+	/**
+	 * Updates a single legacy token in user meta.
+	 *
+	 * @internal
+	 *
+	 * @since 5.6.0-dev.1
+	 *
+	 * @param int $user_id WP user ID
+	 * @param SV_WC_Payment_Gateway_Payment_Token $token token to update
+	 * @param string|null $environment_id optional environment ID, defaults to plugin current environment
+	 * @return string|int updated user meta ID
```

Comment:
`update_user_meta()` can return `int|bool` and not `string` IIRC 

does this method need to return anything though?

Reply:
This method is heavily "inspired" in the existing `update_tokens`, so I thought I could keep the return type, for consistency.